packages/@react-aria/slider/intl/el-GR.json

```
@@ -0,0 +1,4 @@
+{
+  "Maximum": "Το μέγιστο",
+  "Minimum": "Ελάχιστο"
```

Ah, just noticed but these should be lowercase or it breaks when switching to greece locale
```suggestion
  "maximum": "Το μέγιστο",
  "minimum": "Ελάχιστο"
```

---

Not sure why they got switched, but you're right.